specification/riskiq/Easm/models.tsp

```diff
@@ -1671,9 +2034,121 @@ model MarkParameter {
+@Versioning.added(Versions.v2024_03_01_preview)
+model RecentOnlyParameter {
+  @query("recentOnly")
+  @doc("If it's recent only.")
+  recentOnly?: boolean;
+}
+
 @doc("An expression on the resource type that selects the resources to be returned.")
 model RequiredFilterParameter {
   @doc("An expression on the resource type that selects the resources to be returned.")
   @query("filter")
   filter: string;
 }
+
+@resource("cisaCves")
+@doc("cisa cve in a given workspace.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model CisaCveResult {
+  @key("cveId")
+  @visibility("read")
+  @doc("The CVE ID of the vulnerability in the format CVE-YYYY-NNNN, note that the number portion can have more than 4 digits.")
+  cveId: string;
+
+  @doc("The vendor or project name for the vulnerability.")
+  vendorProject: string;
+
+  @doc("The vulnerability product")
+  product: string;
+
+  @doc("The name of the vulnerability")
+  vulnerabilityName: string;
+
+  @doc("A short description of the vulnerability")
+  shortDescription: string;
+
+  @doc("The required action to address the vulnerability")
+  requiredAction: string;
+
+  @doc("Any additional notes about the vulnerability")
+  notes: string;
+
+  @doc("The date the vulnerability was added to the catalog in the format YYYY-MM-DD")
+  dateAdded: utcDateTime;
+
+  @doc("The date the required action is due in the format YYYY-MM-DD")
+  dueDate: utcDateTime;
+
+  @doc("The date the vulnerability was updated")
+  updatedAt: utcDateTime;
+
+  @doc("The number of assets affected by the vulnerability")
+  count: int64;
+}
+
+@doc("AssetChainRequest containing information needed for the retrieval of the asset chain summary.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model AssetChainRequest {
+  @doc("Asset chain source.")
+  assetChainSource: AssetChainSource;
+
+  @doc("A collection of asset chain source ids.")
+  sourceIds: string[];
+}
+
+@doc("Response for the asset chain summary.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model AssetChainSummaryResult {
+  @doc("A list of asset chain summaries per asset kind")
+  affectedAssetsSummary: AssetChainKindSummaryResult[];
+
+  @doc("A list of disco group summaries")
+  affectedGroupsSummary: DiscoGroupSummaryResult[];
+
+  @doc("The list of exceptions")
+  errors?: ErrorResponse[];
+}
+
+@doc("A list of asset chain summaries per asset kind")
+@Versioning.added(Versions.v2024_03_01_preview)
+model AssetChainKindSummaryResult {
+  @doc("The kind of asset")
+  kind: AssetKind;
+
+  @doc("The amount of assets affected for a given asset kind")
+  affectedCount: int64;
+}
+
+@doc("A list of disco group summaries")
+@Versioning.added(Versions.v2024_03_01_preview)
+model DiscoGroupSummaryResult {
+  @doc("The system generated unique id for the resource.")
+  id: string;
+
+  @doc("The caller provided unique name for the resource.")
+  name: string;
+
+  @doc("The name that can be used for display purposes.")
+  displayName: string;
```

Comment:
Are we sure these properties are always returned?

Reply:
yes by default this display name would be the same as the property name, if it's not specified.